coldfront/core/test_helpers/factories.py

```diff
@@ -1,25 +1,99 @@
+import factory
+from django.contrib.auth.models import User
+from factory import SubFactory
+from factory.fuzzy import FuzzyChoice
+from factory.django import DjangoModelFactory
+from faker import Faker
+from faker.providers import BaseProvider, DynamicProvider
+
 from coldfront.core.field_of_science.models import FieldOfScience
-from coldfront.core.resource.models import ResourceType
-from coldfront.core.project.models import Project, ProjectStatusChoice
+from coldfront.core.resource.models import ResourceType, Resource
+from coldfront.core.project.models import (
+    Project,
+    ProjectUser,
+    ProjectAttribute,
+    ProjectAttributeType,
+    ProjectUserRoleChoice,
+    ProjectUserStatusChoice,
+    ProjectStatusChoice,
+    AttributeType as PAttributeType,
+)
+from coldfront.core.allocation.models import (
+    Allocation,
+    AllocationUser,
+    AllocationUserNote,
+    AllocationAttribute,
+    AllocationStatusChoice,
+    AllocationAttributeType,
+    AllocationChangeRequest,
+    AllocationChangeStatusChoice,
+    AllocationAttributeUsage,
+    AllocationUserStatusChoice,
+    AllocationAttributeChangeRequest,
+    AttributeType as AAttributeType,
+)
 from coldfront.core.grant.models import GrantFundingAgency, GrantStatusChoice
 from coldfront.core.publication.models import PublicationSource
 
-from django.contrib.auth.models import User
 
-from factory.django import DjangoModelFactory
-from factory import SubFactory
+### Default values and Faker provider setup ###
+
+project_status_choice_names = ['New', 'Active', 'Archived']
```

Comment:
The hardcoding of the default values here repeats the default values that are listed in project.utils.add_project_status_choices and project.management.commands.add_default_project_choices. It seems like it could be useful to encode default values for model objects in one location in an app-level config.py file or something similar.